daemons/ipa-kdb/ipa_kdb_principals.c

```diff
@@ -610,14 +615,15 @@ static krb5_error_code ipadb_parse_ldap_entry(krb5_context kcontext,
     lcontext = ipactx->lcontext;
     if (!lcontext) {
         krb5_klog_syslog(LOG_INFO,
-                         "No LDAP connection in ipadb_parse_ldap_entry(); retrying...\n");
+                         "No LDAP connection in ipadb_parse_ldap_entry(); retrying [1]...\n");
```

Comment:
We don't need `lcontext` before calling `ipadb_get_user_auth()`.  So I think we can do away with this entire check, and just have the one you've added on line 635.